content/en/APIUseCases/create_asset.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Use Our API to Create an Asset"
+linkTitle: "Create an Asset"
+weight: 110
+description: >
+  Use this sequence of REST calls to create an asset.
+---
+
+{{% pageinfo %}}
+Use this document with our [Cobalt API documentation](https://docs.cobalt.io) to
+manage assets and pentests.
+{{% /pageinfo %}}
+
+<!-- Future task: set up variables for `YOUR-PERSONAL-API-TOKEN` and
+`YOUR-V2-ORGANIZATION-TOKEN`. May support automated populating of REST calls. -->
+
+This page describes how you can use our API To create an [asset](../../getting-started/glossary/#asset). 
+
+## Create an API Token in the Cobalt UI
+
+1. Navigate to https://app.cobalt.io/settings/api-tokens.
+1. If needed, sign in to the app.
+1. Enter an API Token Name.
+1. Select Generate New Token.
+1. In the modal that appears, you should see your API Token, in the **Secret
+   Token** text box.
+
+Save the API Token. After you exit the modal, you won't see the full token again.
+If you lose it, you may have to revoke the token and start over.
+
+Substitute the API Token for `YOUR-PERSONAL-API-TOKEN` in the REST calls
+described on this page.
+
+## Use the API Token to Authorize Access 
+
+Next, you can use the API Token to authorize access to our other endpoints. Take
+the API Token that you [generated](#create-an-api-token-in-the-cobalt-ui). Substitute that value for `YOUR-PERSONAL-API-TOKEN`:
+
+```
+curl https://api.cobalt.io/orgs \
+     -H "Accept: application/vnd.cobalt.v2+json" \
+     -H "Authorization: Bearer YOUR-PERSONAL-API-TOKEN" \
+     | jq .
+```
+
+{{%expand "Review sample output." %}}
+You should see output similar to:
+
+```
+{
+  "pagination": {
+    "next_page": null,
+    "prev_page": null
+  },
+  "data": [
+    {
+      "resource": {
+        "id": "some_id",
+        "name": "Saxophone - Alto",
+        "token": "YOUR-V2-ORGANIZATION-TOKEN"
+      },
+      "links": {
+        "ui": {
+          "url": "https://api.cobalt.io/links/not_a_jwt_token"
+        }
+      }
+    }
+  ]
+}
+```
+{{% /expand %}}
+  
+From the output, save the value for `token` as your organization token.
+In our API documentation, you'll see this as `YOUR-V2-ORGANIZATION-TOKEN`.
+
+For more information, see our API reference documentation on the
+[organizations](https://docs.cobalt.io/v2/#organizations) `orgs` endpoint.
```

Comment:
[organizations](https://docs.cobalt.io/v2/#organizations) endpoint.

Reply:
The endpoint is the end of the functional part of the URL. Since the callout is to `https://api.cobalt.io/orgs`, the endpoint is `orgs`

(Yes, it'd probably be better if the endpoint were `organizations`.)

Reply:
Makes sense.